app/controllers/organizations_controller.rb

```diff
@@ -68,6 +68,12 @@ def feedback
     @wishes = Wishes::FeedbackFilterer.new(filter_params:, scope: @wishes).call
   end
 
+  def boards
+    @organization = current_user.organization
```

Comment:
I think it's better if we always scope with the subdomain from now on and not the current user. So in this case you can skip this line and do:

`@wishlists = @current_organization.wishlists`

@current organization is defined in the application controller `find_organization `method in the master branch

Reply:
By doing so, as admins for example we will be able to masquerade users in the future and get the proper data, not the one from the Admin account.